pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/RealtimeSegmentImpl.java

```diff
@@ -104,8 +106,9 @@
   private static final int MAX_MULTI_VALUES_PER_ROW = 1000;
 
   public RealtimeSegmentImpl(Schema schema, int capacity, String tableName, String segmentName, String streamName,
-      ServerMetrics serverMetrics, List<String> invertedIndexColumns, int avgMultiValueCount)
-      {
+      ServerMetrics serverMetrics, List<String> invertedIndexColumns, int avgMultiValueCount,
+      List<String> noDictionaryColumns)
```

Comment:
We'd need to refactor this at some point so that all the indexing info (inverted index, no dictionary, etc.) is together instead of adding constructor args, WDYT?